docs/dev-proc/source-code-structure.md

```diff
@@ -65,14 +57,61 @@ Force-pushes to `<platform>/rel_vX.Y.Z`, `<platform>/develop` or
   happen it would be better to have it fixed by original author than the person
   that tries to upstream it some time later.
 
-Force-pushes to  `<platform>/release` branches are unconditionally forbidden.
+Force-pushes to  `dasharo` branches are allowed only for rebasing on new
+coreboot releases, according to the guidelines outlined in the next step.
+
+## Rebasing process
+
+1. Update master/main branch to the recent coreboot upstream tag
+   (and fetch tags as well)
+
+    ```bash
+    git checkout master
+    git fetch upstream
+    git pull upstream `git describe --tags upstream/master --abbrev=0`
+    ```
+
+1. Backup the old `dasharo` branch, for example:
+
+    ```bash
+    git checkout dasharo
+    git checkout -b common-base-4.21
+    ```
+
+1. rebase the current `dasharo` branch on the latest tag, for example:
+
+    ```bash
+    git checkout dasharo
+    git rebase 4.22
+    ```
+
+    If there are problems with rebase, you may create a different branch, e.g.
+    `common-base-4.22` and work until all issues are resolved, then rename the
+    branch back to dasharo
+    ```bash
+    git checkout -b common-base-4.22
+    # resolve issues
+    git branch -d dasharo
+    bit branch -M dasharo
+    ```
+
+1. Push the new `dasharo` branch and a backup of the old branch:
+
+    ```bash
+    git push origin common-base-4.21
```

Comment:
```suggestion
    git push -u origin common-base-4.21
```

The push of the backup shoudl be done immediately in step 1

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020